handlers/Capability/index.js

```diff
@@ -48,16 +48,6 @@ class Capability extends React.Component {
           {capability.slogan}
         </h2>
         <CapabilityHighlights highlights={capabilityHighlights} />
-        <div className="Capability-footer">
```

Comment:
can you remove all of the corresponding css for this as well?
https://github.com/Skookum/skookum.com/blob/master/handlers/Capability/styles.css#L52